doc/docs/modules/ROOT/pages/examples.adoc

```diff
@@ -207,7 +206,6 @@ services:
     hostname: schema_registry
     container_name: schema_registry
     depends_on:
-      - zookeeper
```

Comment:
questo secondo me deve restare, noi stiamo rimuovendo i riferimenti a Zookeeper nel nostro codice in questo caso il container c'è ancora